src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchClientExecutor.cs

```diff
@@ -25,8 +26,11 @@ namespace VirtualClient.Actions
     /// </summary>
     public class SysbenchClientExecutor : SysbenchExecutor
     {
+        private const string PythonCommand = "python3";
```

Comment:
you made a variable here but did not apply it throughout and left loose variable with python3

Reply:
Thanks for the catch. Updated